sdk/analytics/src/main/java/com/klaviyo/analytics/OperationQueue.kt

```diff
@@ -0,0 +1,70 @@
+package com.klaviyo.analytics
+
+import com.klaviyo.analytics.model.Profile
+import com.klaviyo.analytics.networking.ApiClient
+import com.klaviyo.core.KlaviyoException
+import com.klaviyo.core.Registry
+import com.klaviyo.core.config.Clock
+
+internal class OperationQueue {
```

Comment:
Rather than keep adding private methods to the Klaviyo object, figured I'd extract an operations manager class like this

Reply:
This is where we would add an in-memory operations queue for methods invoked prior to initialize, if we do that.

Reply:
Sorry to quibble with the name wonder if ProfileOperationQueue is more appropriate here?

Reply:
Or could this be made into a more general purpose thing?

Reply:
Yeah I see what you mean, right now its just the profile debouncing, and the try/catch inline methods. This name was more generic and forward-looking if we did the in-memory queue of method invoked prior to initializing I'd put it in here too